src/menu.py

```diff
@@ -311,5 +317,3 @@ def connect_channel_views(self, channel_views):
 
     def sizeHint(self):
         return qc.QSize(200, 200)
-
-
```

Comment:
pep8 requires one empty line at EOF.

Reply:
Strange, my editor (VS Code) pretends that there is an empty line at the end of each file, but actually there are several files that don't have it.

Reply:
`black` will take care if this.

Reply:
Fixed it manually before seeing your reply, sorry